kyo-caliban/src/test/scala/kyo/resolversTest.scala

```diff
@@ -21,9 +21,9 @@ class resolversTest extends KyoTest:
             zio.Runtime.default.unsafe.run(v).getOrThrow()
         )
 
-    def testServer() =
+    def testServer(port: Int) =
```

Comment:
I'm not sure these tests are failing in CI because the port isn't available. I imagine it could be the short shutdown timeout. I've different ports for each test as a workaround.